### ISSUE TITLE
9. Images, Art & Video - [E1] 1. Steed, Ryan and Aylin Caliskan. 2021.

### ISSUE THREAD

**Issue by JunsolKim** [UChicago-Computational-Content-Analysis/Readings-Responses-2023#8]
Post questions here for this week's exemplary readings: 1. Steed, Ryan and Aylin Caliskan. 2021. “Image Representations Learned with Unsupervised Pre-Training Contain Human-Like Biases.” ACM FAccT.

**Comment by GabeNicholson** [comment]
When the authors mention the biased results such as "observation that image search results for powerful occupations like CEO systematically under-represented women,"—are they talking about biased as in not 50/50 representation or biased as in, even when conditioning on the relative proportion of women CEO's the algorithm still over represents men? The same question goes for the other points mentioned about science and math attributes. Because it is one thing to be biased in our social sense of using heuristics to make judgments on a population basis, and another thing to be biased in Machine Learning by systematically under or over estimating an outcome conditional on its true distribution.

**Comment by isaduan** [comment]
I do not like (perhaps due to my ignorance of) the conception of bias the authors talk about. To me learning is acquiring 'priors' which is a sort of bias. I think the paper would be more interesting if it investigates not whether AI learns human-like social biases, but what kind of biases AI learns and not learns, when it deviates from 'human social bias' - although I am also equally suspicious about whether there is such a thing as average social bias.

**Comment by chentian418** [comment]
For the unsupervised-learning models, I am still confused about embedded bias: in the absence of labels, how do patterns of bias in the pre-training dataset result in implicitly embedded bias? And how does the co-occurrence of contexts indicates the embedded bias?

**Comment by hsinkengling** [comment]
> I do not like (perhaps due to my ignorance of) the conception of bias the authors talk about. To me learning is acquiring 'priors' which is a sort of bias. I think the paper would be more interesting if it investigates not whether AI learns human-like social biases, but what kind of biases AI learns and not learns, when it deviates from 'human social bias' - although I am also equally suspicious about whether there is such a thing as average social bias.

I suspect that the whole thing about detecing and de-biasing ML models is because they are commonly used in settings where discrimination matteres, job-screening, for example. The audience for this paper also might be very different from traditional social scientists who are used to seeing human biases everywhere.
